backend/organization/views/organization_views.py

```diff
@@ -422,6 +478,27 @@ def patch(self, request, url_slug, format=None):
             "organization_size",
             "get_involved",
         ]
+        if "name" in request.data:
+            if Organization.objects.filter(name__iexact=request.data["name"]).exists():
+                return Response(
+                    {
+                        "message": "Organization with name {} already exists".format(
+                            request.data["name"]
+                        )
+                    },
+                    status=status.HTTP_400_BAD_REQUEST,
+                )
+            if OrganizationTranslation.objects.filter(
+                name_translation__iexact=request.data["name"]
+            ).exists():
+                return Response(
+                    {
+                        "message": "Organization with name {} already exists".format(
+                            request.data["name"]
+                        )
+                    },
+                    status=status.HTTP_400_BAD_REQUEST,
+                )
```

Comment:
@JoshKornfeld I see the same logic multiple times repeated. Why don't you move this logic into a separate utility function and just call the function?

Reply:
Moved reused logic into separated utility functions.